src/internal.c

```diff
@@ -10561,8 +10561,7 @@ static WC_INLINE int GrowOutputBuffer(WOLFSSL* ssl, int size)
 #else
     const byte align = WOLFSSL_GENERAL_ALIGNMENT;
 #endif
-    int newSz = size + ssl->buffers.outputBuffer.idx +
-                ssl->buffers.outputBuffer.length;
+    size_t newSz;
```

Comment:
Don't use size_t unless we have to.
Would word32 do here?

Reply:
not as far as I can figure it, because the whole point is to catch int wrap on the arg to `XMALLOC()`, which is `size_t`.  I think we really need `size_t` specifically, whatever its width.

anyway the old practice around `size_t` went out the window with the pedantic `-Wconversion` cleanup about a year ago.  see ed11669f3c, 4b59588cf3, etc.